packages/edit-site/src/components/global-styles/font-size.js

```diff
@@ -0,0 +1,239 @@
+	};
+
+	const handleFontSizeChange = ( value ) => {
+		// If the user is resetting the value, use the initial value.
+		const newValue = value ?? initialFontSize;
+		updateFontSize( 'size', newValue );
+	};
+
+	const handleFluidChange = ( value ) => {
+		updateFontSize( 'fluid', value );
+	};
+
+	const handleCustomFluidValues = ( value ) => {
+		if ( value ) {
+			// If custom values are use init the values with the current ones.
+			updateFontSize( 'fluid', {
+				min: fontSize.size,
+				max: fontSize.size,
+			} );
+		} else {
+			// If custom fluid values are disabled, set fluid to true.
+			updateFontSize( 'fluid', true );
+		}
+	};
+
+	const handleMinChange = ( value ) => {
+		// If the user is resetting the value, use the initial value.
+		const newValue = value ?? fontSize.size;
+		updateFontSize( 'fluid', { ...fontSize.fluid, min: newValue } );
+	};
+
+	const handleMaxChange = ( value ) => {
+		// If the user is resetting the value, use the initial value.
+		const newValue = value ?? fontSize.size;
+		updateFontSize( 'fluid', { ...fontSize.fluid, max: newValue } );
+	};
+
+	const updateFontSize = ( key, value ) => {
+		const newFontSizes = sizes.map( ( size ) => {
+			if ( size.slug === slug ) {
+				return { ...size, [ key ]: value }; // Create a new object with updated key
+			}
+			return size;
+		} );
+
+		setFontSizes( {
+			...fontSizes,
+			theme: newFontSizes,
+		} );
+	};
+
+	const handleRemoveFontSize = () => {
+		// Navigate to the font sizes list.
+		goBack();
+
+		const newFontSizes = sizes.filter( ( size ) => size.slug !== slug );
+		setFontSizes( {
+			...fontSizes,
+			theme: newFontSizes,
+		} );
+	};
+
+	return (
+		<VStack spacing={ 4 }>
+			<HStack justify="space-between" align="flex-start">
+				<ScreenHeader
+					title={ fontSize.name }
+					description={ sprintf(
+						/* translators: %s: font size preset name. */
+						__( 'Manage the font size %s.' ),
+						fontSize.name
+					) }
+				/>
+				<FlexItem>
+					<Spacer marginTop={ 3 } marginBottom={ 0 } paddingX={ 4 }>
+						<DropdownMenu
+							trigger={
+								<Button
+									size="small"
+									icon={ moreVertical }
+									label={ __( 'Menu' ) }
+								/>
+							}
+						>
+							<DropdownMenuItem onClick={ handleRemoveFontSize }>
+								<DropdownMenuItemLabel>
+									{ __( 'Delete' ) }
+								</DropdownMenuItemLabel>
+							</DropdownMenuItem>
+						</DropdownMenu>
+					</Spacer>
+				</FlexItem>
+			</HStack>
+
+			<View>
+				<Spacer paddingX={ 4 }>
+					<VStack spacing={ 4 }>
+						<FlexItem>
+							<FontSizePreview fontSize={ fontSize } />
+						</FlexItem>
+
+						<InputControl
+							label={ __( 'Name' ) }
+							value={ fontSize.name }
+							onChange={ handleNameChange }
+						/>
+
+						<SizeControl
+							label={ __( 'Size' ) }
+							value={ fontSize.size }
+							withSlider
+							onChange={ handleFontSizeChange }
+							hasUnits={ hasUnits }
+							withReset={ false }
+						/>
```

Comment:
How about adding `__nextHasNoMarginBottom` and `__next40pxDefaultSize`? The same goes for min/max fields.

Reply:
added here: [`55d4b40` (#62328)](https://github.com/WordPress/gutenberg/pull/62328/commits/55d4b40942c2d23772b71631967d797a1ff340f1)